src/rgw/rgw_http_errors.h

```diff
@@ -45,6 +45,9 @@ const static struct rgw_http_errors RGW_HTTP_ERRORS[] = {
     { ERR_USER_SUSPENDED, 403, "UserSuspended" },
     { ERR_REQUEST_TIME_SKEWED, 403, "RequestTimeTooSkewed" ,"The difference between the request time and the server's time is too large."},
     { ERR_QUOTA_EXCEEDED, 403, "QuotaExceeded" },
+    { ERR_BAD_RENAME_REQ, 403, "Rename request must have object name, new object name and the HTTP method should be PUT." },
```

Comment:
this structure should have 4 entried instead of 3
3rd is the error code and 4th one is error message. Right now error message will be displayed as error code